Makefile

```diff
@@ -106,17 +106,35 @@ service-facs-si-ps-trims-c1234-stop:
 service-facs-si-ps-trims-c1234-start:
 	cd services; docker stack deploy -c docker-stack-si-ps-trims-c1234.yml facs-si-ps-trims-c1234
 
-service-facs-as-ap-1-stop:
-	cd services; docker stack rm facs-as-ap-1
+service-facs-si-ap-sofb-stop:
+	cd services; docker stack rm facs-si-ap-sofb
+
+service-facs-si-ap-sofb-start:
+	cd services; docker stack deploy -c docker-stack-si-ap-sofb.yml facs-si-ap-sofb
+
+service-facs-as-ap-posang-stop:
+	cd services; docker stack rm facs-as-ap-posang
+
+service-facs-as-ap-posang-start:
+	cd services; docker stack deploy -c docker-stack-as-ap-posang.yml facs-as-ap-posang
+
+service-facs-as-ap-opticscorr-stop:
+	cd services; docker stack rm facs-as-ap-opticscorr
+
+service-facs-as-ap-opticscorr-start:
+	cd services; docker stack deploy -c docker-stack-as-ap-opticscorr.yml facs-as-ap-opticscorr
 
-service-facs-as-ap-1-start:
-	cd services; docker stack deploy -c docker-stack-as-ap-1.yml facs-as-ap-1
+service-facs-as-ap-machshift-stop:
```

Comment:
@fernandohds564 , @anacso17 What about moving to 'macshift' ? I find 'machshift' very non-intuitive. I always have 'MAC meeting' in my head, for example.

Reply:
I think we can change it @xresende . I don't remember why we chose `mach` over `mac`.